graphql/server.go

```diff
@@ -344,9 +367,10 @@ func ServeJSONSocket(socket JSONSocket, schema *Schema, makeCtx MakeCtxFunc, log
 	c := &conn{
 		socket: socket,
 
-		schema:  schema,
-		makeCtx: makeCtx,
-		logger:  logger,
+		schema:     schema,
+		makeCtx:    makeCtx,
+		logger:     logger,
+		computeSem: make(semaphore, MaxConcurrentRerunners),
```

Comment:
Could we make the semaphore part of the server struct so that it's shared across computations?